internal/core/graphics_selection.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+#![warn(missing_docs)]
+
+/*!
+This module contains types used for graphics API selection. The types are internal, with the exception of the `wgpu`
```

Comment:
Since this is part of the api, any reason not to move that in api/graphics_selection.rs   (eg, under the `api` module)

Reply:
That's fine with me :)